### PR TITLE
Implements a Pen Augment

### PR DESCRIPTION
**Pull request by Ben10083**
Title

A new pen augment which can change color and use cursive.

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_augments.dm [hunk 1]
```diff
@@ -61,6 +61,19 @@
 	augs["retractable lighter, left hand"] = /obj/item/organ/internal/augment/tool/combitool/lighter/left
 	gear_tweaks += new /datum/gear_tweak/path(augs)
 
+/datum/gear/augment/tool/pen
```
Comment (mikomyazaki) [inline code/modules/client/preference_setup/loadout/loadout_augments.dm:64]: To follow the pattern of the other `/datum/gear/augment`s this should be `/datum/gear/augment/pen`.

File: code/modules/organs/subtypes/augment.dm [hunk 2]
```diff
@@ -144,6 +144,18 @@
 	parent_organ = BP_L_HAND
 	aug_slot = slot_l_hand
 
+/obj/item/organ/internal/augment/tool/pen
```
Comment (mikomyazaki) [inline code/modules/organs/subtypes/augment.dm:147]: This type will have `parent_organ = BP_CHEST` due to not overriding that from `/obj/item/organ/internal/augment`

File: code/modules/paperwork/pen.dm [hunk 3]
```diff
@@ -310,3 +310,40 @@ Pen exclusive commands
 /obj/item/pen/crayon/Initialize()
 	. = ..()
 	name = "[colourName] crayon"
+
+/*
+ * Augment Pens
+ */
+
+/obj/item/pen/augment
+	name = "Integrated pen"
```
Comment (mikomyazaki) [inline code/modules/paperwork/pen.dm:319]:
```suggestion
	name = "integrated pen"
```

**Review by mikomyazaki — CHANGES_REQUESTED** [review]
Also it is currently possible to change the hand the pen is in, if you click on it with the other hand while the pen is extended.

**Comment by Ben10083** [comment]
> Also it is currently possible to change the hand the pen is in, if you click on it with the other hand while the pen is extended.

Turns out this issue extends to all relevant augments. An issue report has been made(#13878)

**Review by mikomyazaki — APPROVED** [review]

File: code/modules/paperwork/paper.dm [hunk 4]
```diff
@@ -462,7 +462,7 @@
 		if(istype(i, /obj/item/pen/crayon))
 			iscrayon = TRUE
 
-		if(istype(i, /obj/item/pen/fountain))
+		if(istype(i, /obj/item/pen/fountain) || istype(i, /obj/item/pen/augment))
 			var/obj/item/pen/fountain/f = i
```
Comment (NonQueueingMatt) [inline code/modules/paperwork/paper.dm:466]: This is not necessarily a fountain pen anymore.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**